pkg/controller/storagecluster/reconcile.go

```diff
@@ -1152,3 +1156,129 @@ func getMonCount(nodeCount int) int {
 
 	return count
 }
+
+//ensureJobTemplates returns spec of a job template
```

Comment:
```suggestion
// ensureJobTemplates returns spec of a job template
```

Reply:
reframe this statement as the implementation has been changed since it started